ocean_provider/validation/RBAC.py

```diff
@@ -77,6 +78,16 @@ def get_additional_dids(self):
             for additional_input in additional_inputs
         ]
 
+    def get_data(self):
+        if "data" not in self.request.keys():
+            raise Exception("Data to encrypt is empty.")
+        if not isinstance(self.request["data"], list) and not isinstance(
+            self.request["data"], Asset
+        ):
+            raise Exception("Invalid type of data.")
```

Comment:
I'm not big on raising an Exception here. Is it possible to not have data? Previously we had this option, so it should be still supported? i.e. if there is no data, do not send it?

I would also recommend changing the function name to get_request_data, which is clearer.